Toolhouse.Monitoring/Metrics.cs

```diff
@@ -161,71 +161,11 @@ public static void ObserveHttpRequestDuration(TimeSpan duration)
         /// Instruments a block of code that makes a request to an external API (e.g. via REST / SOAP).
         /// </summary>
         /// <param name="name">Descriptive name of thing being done. Should be a singular noun, e.g. "salesforce".</param>
-        /// <param name="callback">Code to run. Should return true for success, false for failure.</param>
+        /// <param name="makeRequest">Code to run. Should return true for success, false for failure.</param>
+        [Obsolete("Use HttpRequestMetrics.Instrument instead.")]
```

Comment:
Per above, augment this class with the various `Instrument<T>(...)` overloads and update this `[Obsolete]`